.github/workflows/ci.yml

```diff
@@ -786,4 +811,5 @@ jobs:
           cd samples/${DIR}
           echo "==== Verifying that generated unmanaged sources compile ===="
           rm -rf src/main/scala src/test/scala src/it/scala
-          sbt  --client -Dkalix-sdk.version=$SDK_VERSION Test/compile
+          sbt  --client -Dkalix-sdk.version=$SDK_VERSION '++$SCALA_213_VERSION! Test/compile'
```

Comment:
Instead of calling sbt twice, we can use +Test/compile.

Reply:
Above, we have to compile steps for each scala version, here we have only one. Not big issue, but we could eventually merge the two other compile steps into one as well. 

The advantage to have then apart is that we make it easier for us to see if it's a Scala 2.13 or 3 compilation error.

Reply:
yeah, that's the reason I kept those apart.. and I thought this one was less likely to present issues so kept it together. But maybe I can also consider merging the other ones.

Reply:
actually I think I need to call sbt twice here..  or otherwise would need to set `crossScalaVersions := ...` on the same sbt command so that the `+Test` has multiple versions to go through.